docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: '3.5'
+
+services:  
+  ubuntu-cpp:
+    # Usage:
+    #   docker-compose build ubuntu-cpp
+    #   docker-compose run --rm ubuntu-cpp
+    # Parameters:
+    #   ARCH: amd64, arm64v8, s390x, ...
+    #   UBUNTU: 18.04, 20.04
+    image: facebookincubator/velox:amd64-ubuntu-22.04-cpp
+    build:
+      context: .
+      dockerfile: scripts/ubuntu-22.04-cpp.dockerfile
+    volumes:
+      - .:/velox:delegated
+    command: &cpp-command >
```

Comment:
nit: curious, whats cpp-command here ?

Reply:
At the moment I am just defining it as a yaml anchor in case we want to add other images to the docker-compose file so we can reuse the cpp-command without having to write it again. I could remove it at this point as it is not being reused anywhere.